content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+---
+reviewers:
+- dchen1107
+- liggitt
+title: Communication between Nodes and the Control Plane
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+Este documento cataloga las rutas de comunicación entre el servidor {{< glossary_tooltip term_id="kube-apiserver" text="servidor API" >}} y el {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} de Kubernetes. El objetivo es permitir a los usuarios personalizar su instalación para reforzar la configuración de la red de tal manera que el clúster pueda ejecutarse en una red no confiable (o en IPs públicas completas en un proveedor de servicios en la nube).
+
+<!-- body -->
+
+## Nodo a Control Plane
+
+Kubernetes tiene un patrón de API "hub-and-spoke" (centro y radios). Todo uso de la API desde los nodos (o los pods que ejecutan) termina en el servidor API. Ninguno de los otros componentes del plano de control está diseñado para exponer servicios remotos. El servidor API está configurado para escuchar conexiones remotas en un puerto HTTPS seguro (típicamente el 443) con una o más formas de
```

Comment:
_"hub-and-spoke"_ is translated to _"centro y conexiones"_ according to [this Wikipedia article](https://es.wikipedia.org/wiki/Paradigma_de_distribuci%C3%B3n_Spoke-hub).

```suggestion
Kubernetes tiene un patrón de API "hub-and-spoke" (centro y conexiones). Todo uso de la API desde los nodos (o los pods que ejecutan) termina en el servidor API. Ninguno de los otros componentes del plano de control está diseñado para exponer servicios remotos. El servidor API está configurado para escuchar conexiones remotas en un puerto HTTPS seguro (típicamente el 443) con una o más formas de
```